run_meson_command_tests.py

```diff
@@ -123,7 +123,9 @@ def test_meson_installed(self):
         pylibdir = prefix / get_pypath()
         bindir = prefix / get_pybindir()
         pylibdir.mkdir(parents=True)
-        os.environ['PYTHONPATH'] = str(pylibdir)
+        # XXX: join with empty name so it always ends with os.sep otherwise
+        # distutils complains that prefix isn't contained in PYTHONPATH
+        os.environ['PYTHONPATH'] = os.path.join(str(pylibdir), '')
```

Comment:
This might be worth a separate PR, since it fixes the current msys2 failure in Appveyor CI

Reply:
Yes, we hit this in GStreamer too. Seems to be a breakage/change in 3.7.1 (3.7.0 worked fine).

Reply:
sounds good, I'll move it out

Reply:
I've opened #4509 for this